### PR TITLE
Add /intents service

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: web/intents/intents.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	intent.SetID("")
+	intent.SetRev("")
+	intent.Services = nil
+	if err = intent.Save(instance); err != nil {
+		return wrapIntentsError(err)
+	}
+	if err = intent.FillServices(instance); err != nil {
+		return wrapIntentsError(err)
+	}
+	if err = intent.Save(instance); err != nil {
```
Comment (aenario) [inline web/intents/intents.go]: Why save in 2 steps?

File: pkg/intents/intents.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+		return ret
+	}
+	return u.String() + buf.String()
+}
+
+// FillServices looks at all the application that can answer this intent
+// and save them in the services field
+func (in *Intent) FillServices(instance *instance.Instance) error {
+	var res []apps.WebappManifest
+	err := couchdb.GetAllDocs(instance, consts.Apps, &couchdb.AllDocsRequest{}, &res)
```
Comment (aenario) [inline pkg/intents/intents.go]: We could use a couchdb view to find intent-corresponding apps.

File: web/intents/intents.go (hunk 1 above)
Comment (nono) [inline web/intents/intents.go]: We need an id for building the href of services, and we need later to persist the services

File: pkg/intents/intents.go (hunk 2 above)
Comment (nono) [inline pkg/intents/intents.go]: Not really, we can do a first filtering with a couchdb view, but not have the intents because of the possible jokers (`*`)

**Pull request merged**